examples/features/load_balancing/client/main.go

```diff
@@ -123,4 +141,87 @@ func (*exampleResolver) Close()                                  {}
 
 func init() {
 	resolver.Register(&exampleResolverBuilder{})
+	balancer.Register(&customRoundRobinBuilder{})
```

Comment:
Maybe put the custom LB policy into its own file (or even its own sub-package), since that's how we expect it to be used typically?

Reply:
Done.